setup.py

```diff
@@ -39,6 +30,17 @@
       author_email='info@precice.org',
       license='LGPL-3.0',
       packages=['fenicsprecice'],
-      install_requires=['pyprecice>=3.0.0.0', 'scipy', 'numpy>=1.13.3, <2', 'mpi4py'],
+      setup_requires=['mpi4py<4'],
+      install_requires=['pyprecice>=3.0.0.0', 'scipy', 'numpy>=1.13.3, <2', 'mpi4py<4'],
       test_suite='tests',
       zip_safe=False)
+
+try:
+    from fenics import *
+except ModuleNotFoundError:
+    warnings.warn("No FEniCS installation found on system. Please install FEniCS and check the installation.\n\n"
+                  "You can check this by running the command\n\n"
+                  "python3 -c 'from fenics import *'\n\n"
+                  "Please check https://fenicsproject.org/download/ for installation guidance.\n"
+                  "The installation will continue, but please be aware that your installed version of the "
+                  "fenics-adapter might not work as expected.")
```

Comment:
See above. Try not to fix a broken FEniCS installation with this `setup.py` then you can also move this up to the original position. Note: We might remove this warning very soon anyways.